### ISSUE TITLE
Q: What is BEDPE?

### ISSUE THREAD

**Issue by BeyondTheProof** [macs3-project/MACS#464]
Hi,

I'm looking over the documentation for `callpeak` and it states, about the BEDPE format: `Please note, this is NOT the same format used by BEDTOOLS, and the BEDTOOLS version of BEDPE is actually not in a standard BED format.`

Going to Google and typing in BEDPE format leads to https://bedtools.readthedocs.io/en/latest/content/general-usage.html. It seems BEDTOOLS developed the BEDPE format as a different format from simple BED.

When using MACS to convert to "BEDPE" format, as suggested on the `callpeak` doc, `macs2 randsample -i align.bam -f BAMPE -p 100 -o align.bedpe`, it returns a simple BED format, not BEDPE format. There are three columns: `chr start end`

My question is: What is BEDPE according to MACS? Is it just the same thing as bed, just with `chrom start_of_fragment end_of_fragment`? If so, I suggest changing the "BEDPE" format you refer to to a different name, such as BEDFRAG or something like that.

Thanks!
Artur

**Comment by BeyondTheProof** [comment]
Answered my own question. Clearly states in the document: `which only contains the first three columns defining the chromosome name, left and right position of the fragment from Paired-end sequencing`

Regardless, I still suggest changing the name BEDPE to BEDFRAG or something of the sort :)

**Comment by kalavattam** [comment]
Hi @taoliu,

First, I want to express my gratitude for your dedication to developing and maintaining such a valuable tool as MACS. It's been instrumental in advancing our research.

I recently encountered a bit of confusion regarding the terminology used for BED output from paired-end sequencing data. Specifically, the term "BEDPE" seems to conflict with an already established format closely associated with Hi-C data, as detailed [here](https://bedtools.readthedocs.io/en/latest/content/general-usage.html#bedpe-format).

During my use of `macs3 filterdup`, I anticipated an output in the conventional BEDPE format but received a three-column BED-like file instead. This discrepancy wasn't immediately clear from the `macs3 filterdup` documentation, and I didn't find a specific definition of BEDPE as used by MACS in [the README Markdown file](https://github.com/macs3-project/MACS/blob/master/README.md). I did, however, find it in [the repo's callpeak Markdown file](https://github.com/macs3-project/MACS/blob/1b92238837de870019aacfeae1fb38f407f77903/docs/callpeak.md?plain=1#L93).

For clarity and to help future users, it might be beneficial to explicitly define the BEDPE format as understood by MACS, possibly in the main README or within the tool's documentation text for the various functions. This small adjustment could enhance user experience and prevent similar confusion.

Thank you again for your incredible work on this project. Your efforts are greatly appreciated by the scientific community.

Best,
Kris
